src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -47,4 +47,22 @@ private function getLanguageClasses(): array {
 		return [ 'language-' . htmlspecialchars( $this->language ) ];
 	}
 
+	/**
+	 * @return (string|string[])[]
+	 */
+	private function getWrapperAttributes( string $contentUrl ): array {
+		$attributes = [];
+
+		$attributes['class'] = $this->getWrapperClasses();
+
+		$attributes['data-toolbar-order'] = 'copy-to-clipboard';
```

Comment:
What is this bit attribute for?

Reply:
Although we do not need ordering if the Edit button is not being included. But how the plugin works is that it includes all the registered buttons if this variable is not defined, which in case it is not a bitbucket code, we do not want to show the Edit button. Hence, I have by default defined to only include the Copy to Clipboard button.